src/python/WMCore/Database/DBCore.py

```diff
@@ -156,15 +163,28 @@ def processData(self, sqlstmt, binds={}, conn=None,
                 #Run single SQL statement for a list of binds - use execute_many()
                 if not transaction:
                     trans = connection.begin()
-                while(len(binds) > self.maxBindsPerQuery):
-                    result.extend(self.processData(sqlstmt, binds[:self.maxBindsPerQuery],
-                                                   conn=connection, transaction=True,
-                                                   returnCursor=returnCursor))
-                    binds = binds[self.maxBindsPerQuery:]
 
-                for i in sqlstmt:
-                    result.extend(self.executemanybinds(i, binds, connection=connection,
-                                                        returnCursor=returnCursor))
+                s = sqlstmt[0]
+
+                if s.lower().endswith('select', 0, 6) and returnCursor:
+                    iterableFlag = True
+                else:
+                    iterableFlag = False
+
+                if iterableFlag:
```

Comment:
Why not remove this variable and just do:

```
if s.lower().endswith('select', 0, 6) and returnCursor:
```

Reply:
I will fix this